server/portal/apps/webhooks/models.py

```diff
@@ -3,7 +3,7 @@
 from django.db import models
 from django.conf import settings
 from datetime import datetime
-from portal.apps.webhooks.fields import JSONField
+from portal.utils.fields import JSONField
```

Comment:
Does this change require a migration?

Reply:
Ah there is a [migration](https://github.com/TACC/Core-Portal/blob/a37737e8d8ac8eeb6be9909d4eae38a94702ba2b/server/portal/apps/webhooks/migrations/0002_auto_20221221_2114.py)!

Reply:
Ah don't know how I missed that 👍